sdk/attestation/azure-security-attestation/azure/security/attestation/_administration_client.py

```diff
@@ -4,316 +4,736 @@
+        )
+        token = AttestationToken(
+            token=cert_response.token,
+            body_type=GeneratedPolicyCertificatesModificationResult,
+        )
+
+        if options.get("validate_token", True):
+            token._validate_token(  # pylint: disable = protected-access
+                self._get_signers(**kwargs), **options
+            )  # pylint: disable = protected-access
+        return (
+            AttestationPolicyCertificateResult._from_generated(  # pylint: disable = protected-access
+                token._get_body()  # pylint: disable = protected-access
+            ),
+            token,
+        )
 
     @distributed_trace
-    def remove_policy_management_certificate(self, certificate_to_add, signing_key, **kwargs):
-        #type:(bytes, AttestationSigningKey, **Any) -> AttestationResponse[PolicyCertificatesModificationResult]
-        """ Removes a new policy management certificate to the set of policy management certificates for the instance.
+    def remove_policy_management_certificate(self, *args, **kwargs):
+        # type: (*str, **Any) -> Tuple[AttestationPolicyCertificateResult, AttestationToken]
+        # pylint: disable = line-too-long
+        """Removes a policy management certificate from the set of policy management certificates for the instance.
 
-        :param bytes certificate_to_add: DER encoded X.509 certificate to add to 
+        :param str certificate_to_remove: PEM encoded X.509 certificate to remove from
             the list of attestation policy management certificates.
-        :param signing_key: Signing Key representing one of 
+        :keyword  str signing_key: PEM encoded signing Key representing the key
+            associated with one of the *existing* attestation signing certificates.
+        :keyword str signing_certificate: PEM encoded signing certificate which is one of
             the *existing* attestation signing certificates.
-        :type signing_key: azure.security.attestation.AttestationSigningKey
-        :return: Attestation service response 
-            encapsulating a list of DER encoded X.509 certificate chains.
-        :rtype: azure.security.attestation.AttestationResponse[azure.security.attestation.PolicyCertificatesModificationResult]
-
-        The :class:`PolicyCertificatesModificationResult` response to the 
+        :keyword bool validate_token: if True, validate the token, otherwise return the token unvalidated.
+        :keyword validation_callback: Function callback to allow clients to perform custom validation of the token.
+            if the token is invalid, the `validation_callback` function should throw
+            an exception.
+        :paramtype validation_callback: ~typing.Callable[[~azure.security.attestation.AttestationToken, ~azure.security.attestation.AttestationSigner], None]
+        :keyword bool validate_signature: if True, validate the signature of the token being validated.
```

Comment:
The casing of parameter descriptions is inconsistent (sometimes they lead with a capital letter, sometimes not). It's not a big deal, but it's best to decide on one or the other -- I'd lean towards capital-leading descriptions. This comment applies throughout the class' docstrings
```suggestion
        :keyword bool validate_signature: If True, validate the signature of the token being validated.
```

Reply:
For reasons I cannot explain, it appears this only applies to "if" - most of them are cut&paste errors; but I've scrubbed for all of them.